Web/Models/Entities/Post.php

```diff
@@ -153,10 +158,11 @@ function setLike(bool $liked, User $user): void
             parent::setLike($liked, $user);
         }
 
-        foreach($this->getChildren() as $attachment) {
-            if($attachment instanceof Post)
-                $attachment->setLike($liked, $user);
-        }
+        if($depth < ovkGetQuirk("wall.repost-liking-recursion-limit"))
+            foreach($this->getChildren() as $attachment) {
+                if($attachment instanceof Post)
+                    $attachment->setLikeRecurse($liked, $user, $depth + 1);
+            }
```

Comment:
Would you kindly remove redundant "{" and "}"?

Reply:
Done